docs/pbc/all/dynamic-multistore/202404.0/base-shop/dynamic-multistore-feature-overview.md

```diff
@@ -80,3 +80,84 @@ Normal storefront APIs can be used to fetch data from stores using the Store HTT
 The following image shows the ability to call API endpoints with the exemplary "SECOND" store selected in the API call itself.
 
 ![storefront-api](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/storefront-api.png)
+
+## Adding of a new store via data import
+
+Example for DE store configuration:
+
+**data/import/common/{REGION}/store.csv**
+
+```csv
+name
+DE
+AT
+```
+
+| Column     | REQUIRED | Data Type | Data Example | Data Explanation |
+|------------| --- | --- | --- | --- |
+|name        |mandatory |string | DE | Define store name. |
+
+Configure a new store as described here: [Import configuration for store](/docs/pbc/all/dynamic-multistore/202311.0/base-shop/install-and-upgrade/install-features/install-dynamic-multistore.html#import-data)
+
+
+Adding a new store via the Back Office is easier and faster, but in this case, you should add this store across all environments.
+Data import allows you to add and configure a new store once and deploy it across all environments.
+
+Note that when you add a new store you should also assign all store-related entities to it for enabling them for customers:
+- Products
+- Categories
+- CMS entities
+- Prices
+- etc.
+
+To avoid manual assignment of entities in Back Office to the store you can use the data import for store-related entities as well.
+
+See: [Import minimum set of data for store](/docs/pbc/all/dynamic-multistore/202404.0/base-shop/import-stores.html)
+
+
+## What is changed when DMS is enabled?
+
+DMS enables the following changes to the project:
+
+In represented examples below there are two regions: EU and US.
+
+EU region has two stores: DE and AT. US region has one store: US.
+
+- Urls for Yves, Back Office, Merchant Portal and Glue contain region instead of store name.
+  
+  Example for local environment: https://backoffice.eu.mysprykershop.com instead of https://backoffice.de.mysprykershop.com 
+  ![local-dashboard-non-dms]()
```

Comment:
<img width="497" alt="local-dashboard-non-dms" src="https://github.com/user-attachments/assets/5bb9626a-b2e5-4a27-831d-a68956efa519">

Reply:
For the links, I'm not sure we need images here. Having only `Example for local environment: https://backoffice.eu.mysprykershop.com instead of https://backoffice.de.mysprykershop.com` as an example would be enough